deploy/AzureAppServices/Datadog.Trace.AzureAppServices.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Datadog.Trace.AzureAppServices</id>
+    <version>1.10.0</version>
+    <title>Datadog APM - .NET Tracer</title>
+    <authors>Robert Glickman;Lucas Pimentel-Ordyna;Colin Higgins</authors>
+    <owners>Lucas Pimentel-Ordyna</owners>
+    <license type="expression">Apache-2.0</license>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+      Installer for Datadog's .NET Tracer for APM.
+      Required configuration parameters are: DD_API_KEY.
+	    Optional configuration parameters are: DD_AGENT_HOST, DD_ENV and DD_SERVICE_NAME
```

Comment:
I think we should only mention that `DD_API_KEY` is required and just link to the official docs for the rest.